packages/hop-node/src/constants/constants.ts

```diff
@@ -48,7 +48,7 @@ export const SettlementGasLimitPerTx: Record<string, number> = {
   arbitrum: 59105
 }
 
-const SecondsInDay = 86400
+export const SecondsInDay = 86400
```

Comment:
small suggestion on naming: `SecondsInDay` -> `OneDaySeconds` (since we use `OneHourMs`, `OneHourSeconds`, etc)

Reply:
Updated in 3b3f8a3c0464023e8ac6bd8ec3a023029a169228